soccer/gameplay/plays/offense/BasicOffensiveScramble.py

```diff
@@ -0,0 +1,73 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import evaluation
+import situational_play_selection
+import tactics.coordinated_pass
+import skills.move
+import skills.capture
+
+class BasicOffensiveScramble(standard_play.StandardPlay):
+
+    _situationList = [
+        situational_play_selection.SituationalPlaySelector.Situation.OFFENSIVE_SCRAMBLE
+    ] # yapf: disable
+
+    class State(enum.Enum):
+        get_ball = 1, 'Start the play'
+
+    def __init__(self):
+        super().__init__(continuous = False)
+
+        for s in BasicOffensiveScramble.State:
+            self.add_state(s, behavior.Behavior.State.running)
+
+        self.add_transition(behavior.Behavior.State.start,
+            BasicOffensiveScramble.State.get_ball, lambda: True, 'Immidiatley')
+
+        self.add_transition(BasicOffensiveScramble.State.get_ball,
+            behavior.Behavior.State.completed,
+            lambda: self.subbehavior_with_name('Capture ball').state == behavior.
```

Comment:
There's a `is_done_running` function that you can call on the subbehavior to check if it's completed or failed (or cancelled? Whatever that means)

https://github.com/RoboJackets/robocup-software/blob/staging/soccer/gameplay/behavior.py#L38